src/app/pages/wallets/wallet-details/transaction-editor/pages-wallet-transaction-editor.component.html

```diff
@@ -0,0 +1,36 @@
+<form [formGroup]='form'>
+  <h1 mat-dialog-title>{{ data ? "Update Transaction" : "Create Transaction" }}</h1>
+  <mat-form-field appearance="fill">
```

Comment:
All those inputs are now in one row. Display them pls in column one under another. With some margins between them.